src/NuGet.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <!-- NOTE: Leave this file here and keep it in sync with list in dir.props. -->
+  <!-- The command-line doesn't need it, but the IDE does.                    -->
```

Comment:
I think I would actually make this a little more specific and say the command line tool doesn't correctly support this yet but we want to use the config file for both in the future.